src/metadata/metadata_hash.c

```diff
@@ -563,6 +563,7 @@ int ocf_metadata_hash_init(struct ocf_cache *cache,
 	for (i = 0; i < OCF_IO_CLASS_MAX + 1; i++) {
 		cache->user_parts[i].config = &part_config[i];
 		cache->user_parts[i].runtime = &part_runtime[i];
+		env_spinlock_init(&cache->user_parts[i].runtime->reserved_clines_barrier);
```

Comment:
it' suprisning for me to see a spin lock variable named "barrier". How about reserved_clines_lock?